### DISCUSSION TITLE
Neofetch

### DISCUSSION THREAD

**Discussion by sudobrat** [prasanthrangan/hyprdots#103] [Q&A]
the neofetch config is great but I also want that ascii arch A logo . please guide what I can do

**Comment by T-Crypt** [comment]
You need to take a look at the github wiki for Neofetch customizing. 

You will need to specify the ASCII output for the arch logo. 

neofetch --ascii_distro archlinux  
https://github.com/dylanaraps/neofetch/wiki/Image-Backends

Take a look at line #698 in the config for neofetch in the /.config directory 

https://github.com/prasanthrangan/hyprdots/blob/main/Configs/.config/neofetch/config.conf
